docs/migrations/v6-to-v7.md

```diff
@@ -13,7 +13,31 @@ There are four sections based on the four potential user groups of this document
 
 ## Chains
 
-- No relevant changes were made in this release.
+Chains will perform automatic migrations to remove a localhost client, if it exists and to migrate the solomachine to the v3 of the protobuf definition. 
+
+An optional upgrade handler has been added to prune expired tendermint consensus states.
```

Comment:
```suggestion
An optional upgrade handler has been added to prune expired tendermint consensus states. It may be used during any upgrade (not just to v7)
```